src/main/kotlin/docs/04_Drawing_basics/C08_DrawingPrimitivesBatched.kt

```diff
@@ -0,0 +1,275 @@
+@file:Suppress("UNUSED_EXPRESSION")
+
+package docs.`04_Drawing_basics`
+
+import org.openrndr.application
+import org.openrndr.color.ColorRGBa
+import org.openrndr.color.rgb
+import org.openrndr.dokgen.annotations.*
+import org.openrndr.draw.circleBatch
+import org.openrndr.draw.rectangleBatch
+import org.openrndr.extensions.SingleScreenshot
+import org.openrndr.shape.Circle
+import org.openrndr.extra.noise.Random
+import org.openrndr.math.Polar
+import org.openrndr.math.Vector2
+import org.openrndr.shape.Rectangle
+
+fun main() {
+    @Text """
+    # Drawing primitives batched
+
+    OPENRNDR provides special draw APIs for drawing many circles, rectangles or 
+    points at once by doing something called "batching".
+
+    This technique performs much faster because the rendering 
+    time is tied to the number of calls we do on the GPU. It takes less 
+    time to send one large piece of data from CPU to GPU than sending many 
+    small pieces. Using this approach can be beneficial when drawing hundreds
+    or thousands of elements.
+    
+    """
+
+    @Text """
+    ## Batched circles
+
+    This example makes use of `Circle` (a class with properties like 
+    `center` and `radius` and some useful methods), not to be confused with 
+    `drawer.circle()` (a method that draws pixels on the screen). 
+     It is possible to construct a `Circle` using `center` and `radius` but 
+     also with two or three `Vector2` points.
```

Comment:
I feel this sentence doesn't really tell the user what difference between the two constructors is. How about...

> It is possible to construct a `Circle` by providing a `center` position and a `radius` but 
     also with two or three `Vector2` points that are used for deriving the circumference of a `Circle`.

Reply:
Much better, thanks!